docs/slack.rst

```diff
@@ -259,22 +259,48 @@ In addition to the guidance that applies to all participants, we expect self-pro
   Clearly explain what you are promoting, why it matters to documentarians, and your involvement.
   Be up front about paywalls, registration requirements, referral benefits, or commissions.
   If you're running a survey, say when and how the results will be published.
+
 - **Give more than you take.**
   Make self-promotion a reasonable proportion of your participation in the community.
   Too much self-promotion creates distrust and hostility.
   If you're not sure if it's too much, aim to help others 10 times for every time you self-promote.
+
 - **Post publicly instead of sending direct messages (DMs)**.
   A DM is like a home and you're like a vampire: you're not allowed in without an invitation.
   It can be tempting to DM people, but please respect their inboxes and post in public channels unless invited to DM.
+
 - **Read the room.**
   The Write the Docs Slack is informal and conversational; high-pressure sales aren't welcome.
   If people aren't interested in what you're offering, don't take it as a license to become louder and more aggressive.
   If you're asked to stop, then stop.
+
 - **Treat the community like friends, not customers.**
   Posts that resemble infomercials are harder for the community to engage with and for the moderation team to distinguish from spam.
   If you post about things that aren't relevant to the documentation field or the channel you're posting in, moderators might delete your messages.
   Make sure conversations stay relevant and not sales-focused, so you can have the chance to talk about work you're excited about.
 
+Here are some examples of common self-promotion situations, and some guidelines on how to self-promote responsibly:
```

Comment:
This is a large enough addition to section that takes up the full vertical height of the screen (at least on my screen), which makes it somewhat harder to skim. Maybe this section could be broken in three:

- The first bit with the actual guidelines, immediately following the heading
- A new subhead (h4) for the examples
- A new subhead (h4) for the self-promotion channel recommendations

(Or if that doesn't suit, I can suggest cutting down the text a bit.)

Reply:
Done!